app/app.py

```diff
@@ -42,6 +42,9 @@ def get_application(settings: Settings, drop_db: bool = False) -> FastAPI:
     if not os.path.exists("data/ics/"):
         os.makedirs("data/ics/")
 
+    if not os.path.exists("data/core/"):
```

Comment:
Is there any policy on when getting rid of these backup email files ?

Reply:
I don't think so.
For now we just keep a trace of all email migrations.